packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -197,6 +197,9 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     deleteCompletedTxs() {
       return statusManager.deleteCompletedTxs();
     },
+    getChainHub() {
```

Comment:
food for thought: `withOrchestration` could automatically provide such methods, wrapping the `contract` return value. E.g. instead of `creatorFacet` it could return `orchestrationFacet` or something.

probably too much magic as yet